hawkular-inventory-rest-api/pom.xml

```diff
@@ -331,7 +359,7 @@
                   <basePath>/hawkular/inventory</basePath>
                   <info>
                     <title>Hawkular Inventory</title>
-                    <version>0.1</version>
+                    <version>${project.version}</version>
```

Comment:
hmm, should the REST api version be the same as the project version? It's questionable.